tests/test_data/test_app_dir_struct/data/DemoApplicationDataSource/instances/package.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "instances",
   "type": "CORE:Package",
+  "isRoot": false,
   "_meta_": {
-    "type": "CORE:Meta",
+    "type": "dmss://system/SIMOS/Meta",
```

Comment:
This change should not be needed.
If the alias does not work for "Meta", we should fix that.